test/lit/passes/inlining-gc.wast

```diff
@@ -0,0 +1,45 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; RUN: wasm-opt %s --inlining --enable-gc-nn-locals -all -S -o - | filecheck %s
+
+(module
+ ;; CHECK:      (func $caller-n
+ ;; CHECK-NEXT:  (local $0 funcref)
+ ;; CHECK-NEXT:  (block $__inlined_func$target-n
+ ;; CHECK-NEXT:   (local.set $0
+ ;; CHECK-NEXT:    (ref.null func)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:   (nop)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $caller-n
```

Comment:
Nit: How about `n`  -> `nullable` and `nn` -> `non-nullable` in function names? Not a big deal but will be slightly easier to read later.

Reply:
Done.